src/dvclive/live.py

```diff
@@ -571,7 +558,8 @@ def make_report(self):
 
     @catch_and_warn(DvcException, logger)
     def make_dvcyaml(self):
-        make_dvcyaml(self)
+        if self.dvc_file:
```

Comment:
what is the case for this?

Reply:
I mean when can it be `None`?

Reply:
this is the only (?) unresolved question? Approved, since it doesn't look anything major ... 


good stuff, Dave. thanks!

Reply:
Sorry, missed this one. Good catch. Maybe this came from an earlier iteration. I dropped this condition in the latest commit.